mautrix_instagram/portal.py

```diff
@@ -1265,7 +1267,10 @@ async def _handle_instagram_reactions(
                 continue
             puppet = await p.Puppet.get_by_pk(new_reaction.sender_id)
             intent = puppet.intent_for(self)
-            reaction_event_id = await intent.react(self.mxid, message.mxid, new_reaction.emoji)
+            if timestamp:
+                reaction_event_id = await intent.react(self.mxid, message.mxid, new_reaction.emoji, timestamp=timestamp)
+            else:
+                reaction_event_id = await intent.react(self.mxid, message.mxid, new_reaction.emoji)
```

Comment:
does this work if you just do 

```suggestion
            reaction_event_id = await intent.react(self.mxid, message.mxid, new_reaction.emoji, timestamp=timestamp)
```

If it's `None`, does it just exclude the key?

Reply:
I guess it does ;)